### PR TITLE
Added README.md under lesson_00

### PR DESCRIPTION
**Pull request by cibezim**
I have added my README.md to my new folder chukwumaibezim under lesson_00 from my branch cibezm. I do not want my README published publicly.

### CODE REVIEW

**Review by anthonydmays — CHANGES_REQUESTED** [review]
Please fix your html changes or remove and add via a separate PR since only the markdown version is due initially. Up to you whichever you choose.

File: lesson_00/chukwumaibezim/README.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (cibezim) [reply lesson_00/chukwumaibezim/README.html]: deleted html file

**Review by cibezim — COMMENTED** [review]
all done

File: lesson_00/chukwumaibezim/README.md
Comment (anthonydmays) [inline lesson_00/chukwumaibezim/README.md:1]: Almost there! Please add something that will help your professional colleagues understand how to best work with you. Be happy to approve once you've added that.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_00/chukwumaibezim/README.md
Comment (cibezim) [reply lesson_00/chukwumaibezim/README.md:1]: Updated with information

**Review by anthonydmays — APPROVED** [review]
LGTM

**Pull request merged**